### PR TITLE
Change rope fusion default

### PR DESCRIPTION
**Pull request by cuichenx**
# What does this PR do ?

Change rope fusion default as a workaround for #8590

### CODE REVIEW

File: nemo/collections/nlp/models/language_modeling/megatron_base_model.py [hunk 1]
```diff
@@ -471,7 +471,8 @@ def build_transformer_config(self) -> TransformerConfig:
 
         bias_dropout_fusion = self.cfg.get('bias_dropout_add_fusion', True)
 
-        apply_rope_fusion = self.cfg.get('apply_rope_fusion', True)
+        # @chcui default rope fusion to false until #8590 is closed.
+        apply_rope_fusion = self.cfg.get('apply_rope_fusion', False)
```
Comment (ericharper) [inline nemo/collections/nlp/models/language_modeling/megatron_base_model.py:475]: should this be added to megatron_gpt_config.yaml?

Comment (cuichenx) [reply nemo/collections/nlp/models/language_modeling/megatron_base_model.py:475]: good idea, i'll do that

**Review by ericharper — APPROVED** [review]
LGTM. Thanks!

**Comment by ericharper** [comment]
jenkins

**Comment by ericharper** [comment]
Just noticed the base branch needs to be changed to `main`

**Comment by cuichenx** [comment]
opened #8594 instead

**Comment by ericharper** [comment]
closed in favor of #8594

**Pull request closed without merging**